src/components/shared/NetworkSettings.css

```diff
@@ -14,20 +14,10 @@
   flex: 1;
 }
 
-.networkSettingsLabel {
-  display: inline-flex;
-  height: 25px;
-  flex-flow: row nowrap;
-  align-items: center;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  user-select: none;
-}
-
-.networkSettingsSearchbar {
-  padding: 0 5px;
+.networkSettingsSearchField .panelSearchFieldInput {
+  width: 250px;
 }
 
-.networkSettingsSearchField {
-  width: 250px;
+.networkSettingsSearchField .panelSearchFieldIntroduction {
```

Comment:
Optional (no need to re-review if you change it)

On my mac the text has an orphan word that goes to the newline. It might be nice to make it all fit on one, although this isn't really guaranteed on other systems.
<img width="351" alt="image" src="https://user-images.githubusercontent.com/1588648/60624997-40e20200-9dac-11e9-9b55-238d29badb42.png">

Reply:
It looks the same on my Linux system.
But if I widen the introduction box to make all text fit on one line, the label and the input box above will be displaced so that the input will no longer be completely aligned to the right.

The simple fix is to also widen the input a bit, is it fine if I do that?

Reply:
I think it's fine to make the search box wider, like it is in the call tree.